migrations/000028_app_versions_table.down.sql

```diff
@@ -0,0 +1,135 @@
+BEGIN;
+
+SET search_path = public, pg_catalog;
+
+--
+-- Repopulate `apps` data from `app_versions` table
+--
+ALTER TABLE ONLY apps
+    ADD COLUMN IF NOT EXISTS deleted boolean DEFAULT false;
+ALTER TABLE ONLY apps
+    ADD COLUMN IF NOT EXISTS disabled boolean DEFAULT false;
+ALTER TABLE ONLY apps
+    ADD COLUMN IF NOT EXISTS integration_data_id uuid;
+ALTER TABLE ONLY apps
+    ADD COLUMN IF NOT EXISTS integration_date timestamp without time zone;
+ALTER TABLE ONLY apps
+    ADD COLUMN IF NOT EXISTS edited_date timestamp without time zone DEFAULT CURRENT_TIMESTAMP;
+
+UPDATE apps SET (
+    deleted,
+    disabled,
+    integration_data_id,
+    integration_date,
+    edited_date
+) = (
+    SELECT
+        deleted,
+        disabled,
+        integration_data_id,
+        integration_date,
+        edited_date
+    FROM app_versions
+    WHERE app_versions.app_id = apps.id
+    ORDER BY version_order DESC
+    LIMIT 1
+) ON CONFLICT DO NOTHING;
+
+ALTER TABLE ONLY apps
+    ALTER COLUMN deleted SET NOT NULL;
+ALTER TABLE ONLY apps
+    ALTER COLUMN disabled SET NOT NULL;
+ALTER TABLE ONLY apps
+    ALTER COLUMN integration_data_id SET NOT NULL;
+
+ALTER TABLE ONLY app_steps
+    ADD COLUMN IF NOT EXISTS app_id uuid;
+UPDATE app_steps
+    SET app_id = (
+        SELECT app_id
+        FROM app_versions
+        WHERE app_steps.app_version_id = app_versions.id
+        ORDER BY version_order DESC
+        LIMIT 1
+    ) ON CONFLICT DO NOTHING;
+ALTER TABLE ONLY app_steps
+    ALTER COLUMN app_id SET NOT NULL;
+ALTER TABLE ONLY app_steps
+    DROP COLUMN IF EXISTS app_version_id;
+
+--
+-- Foreign Key for `app_id` column in `app_steps` table.
+--
+ALTER TABLE ONLY app_steps
+    ADD CONSTRAINT app_steps_app_id_fkey
+    FOREIGN KEY (app_id)
+    REFERENCES apps(id) ON DELETE CASCADE;
+
+CREATE INDEX IF NOT EXISTS app_steps_app_id ON app_steps(app_id);
+
+--
+-- A view containing the top-level information needed for the app listing
+-- service.
+--
+CREATE OR REPLACE VIEW app_listing AS
```

Comment:
This is just a copy of the current view defined in an earlier migration:
https://github.com/cyverse-de/de-database/blob/f70438ff43838b1d5aeb9072f150d86bf9150535/migrations/000017_views.up.sql#L37-L94